src/App.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import './App.css';
```

Comment:
file imports should all be below module imports....eslint should have caught this, are you using the airbnb style guide?